### ISSUE TITLE
Docker with jsartoolkit5: what changes are needed?

### ISSUE THREAD

**Issue by kalwalt** [kalwalt/jsartoolkit5#12]
Few things about this topic: We saw in this issue #11  that building with emscripten under different OS can be problematic, it is possible to have differents results. To avoid this @ThorstenBux has configured Docker in this branch https://github.com/ThorstenBux/jsartoolkit5/tree/build-with-docker you can follow the instructions in the Readme.md and test it. 
I have some points to underline:

- [x] libjpeg.bc is not requested anymore? it seems that is built by Docker/Emscripten:

```
stdout: create_lib /emsdk_portable/.data/cache/asmjs/ports-builds/libjpeg/libjpeg.bc

stderr: shared:WARNING: Assuming object file output in the absence of `-c`, based on output filename. Please add with `-c` or `-r` to avoid this warning
system_libs:INFO: retrieving port: zlib from https://github.com/emscripten-ports/zlib/archive/version_1.zip
system_libs:INFO: unpacking port: zlib
cache:INFO: generating port: libz.a... (this will be cached in "/emsdk_portable/.data/cache/asmjs/libz.a" for subsequent builds)
cache:INFO:  - ok
system_libs:INFO: retrieving port: libjpeg from https://dl.bintray.com/homebrew/mirror/jpeg-9c.tar.gz
system_libs:INFO: unpacking port: libjpeg
cache:INFO: generating port: libjpeg.bc... (this will be cached in "/emsdk_portable/.data/cache/asmjs/libjpeg.bc" for subsequent builds)
root:INFO: building port: libjpeg
cache:INFO:  - ok

```
i f this is the case we can delete the **libjpeg.bc** in the **emscripten/libjpeg/lib** folder
- [x] libjpeg includes we need them or is included in the Docker/Emscripten?

i will update this list as i can. 

**EDIT:** By emscripten v1.38.31 we don't need to include anymore libjpeg headers and .bc files.

**Comment by kalwalt** [comment]
Found this issue https://github.com/emscripten-core/emscripten/issues/9170 !! :100:

**Comment by kalwalt** [comment]
...and from the mescripten master branch https://github.com/emscripten-core/emscripten/blob/master/tools/ports/libjpeg.py so libjpeg.bc is not needed anymore and probably also the includes because the python script download the headers files if you add the USE_LIBJPEG flag.

**Comment by kalwalt** [comment]
i tested with and without Docker/emscripten and with latest emscripten **1.39.5** using the `-s USE_LIBJPEG` flag it seems that is not needed the libjpeg includes, because emscripten download the source code for libjpeg and built the lib.

**Comment by kalwalt** [comment]
Under linux, emscripten  **1.39.5** download the libjpeg files in the  hidden `.emscripten_ports` folder and store here the **libjpeg.bc** build file.

**Comment by kalwalt** [comment]
libjpeg is in the emscripten tools ports by **v1.38.31** so we can think to change the **makem.js** script.

**Comment by kalwalt** [comment]
Solved with #13
